ansible/roles/test/files/ptftests/sonic.py

```diff
@@ -309,7 +309,7 @@ def check_lag_flaps(self, interface, log_lines, start_time):
         return 0, num_lag_flaps
 
     def parse_lacp(self, output):
-        return six.ensure_str(output).find('Bundled') != -1
```

Comment:
In above line 64, `six.ensure_str` is added, but here same handler is removed. It looks like one of these is not intended way to handle Py2/3. Can you confirm?

Reply:
The `six.ensure_str` call in line 64 will make sure all of the output from SSH calls to the VM are `str` and not `bytes`. The `output` variable in this function is coming from an SSH call, and so will already be a `str` variable.